client/components/upgrades/gsuite/index.jsx

```diff
@@ -7,13 +7,15 @@ import HeaderCake from 'calypso/components/header-cake';
 import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
 import { hasDomainInCart } from 'calypso/lib/cart-values/cart-items';
 import { GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY } from 'calypso/lib/gsuite/constants';
+import useCartKey from 'calypso/my-sites/checkout/use-cart-key';
 import { infoNotice } from 'calypso/state/notices/actions';
 import { getProductsList } from 'calypso/state/products-list/selectors/get-products-list';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import GSuiteUpsellCard from './gsuite-upsell-card';
 
 export default function GSuiteUpgrade( { domain } ) {
-	const { responseCart: cart, addProductsToCart, isLoading } = useShoppingCart();
+	const cartKey = useCartKey();
+	const { responseCart: cart, addProductsToCart, isLoading } = useShoppingCart( cartKey );
```

Comment:
We don't need to update that component as it is no longer used. I've created https://github.com/Automattic/wp-calypso/pull/56268 in order to remove it.